site/zh/blog/new-in-devtools-104/index.md

```diff
@@ -26,162 +25,162 @@ draft: true
 <!-- + 4. Translate all the <img> alt text -->
 <!-- + 5. Update the whats-new.md file -->
 
-<!-- ## Restart frame during debugging {: #restart-frame } -->
+<!-- ## 在调试时进行帧重启 {: #restart-frame  } -->
 
-<!-- The **Restart frame** feature is back! You can re-run the preceding code when paused somewhere in a function. Previously, this feature was deprecated and removed in Chrome 92 due to stability issues.  -->
+<!-- **帧重启** 功能回归了! 你可以在函数中途从头开始重启当前代码行之前的代码。这个功能曾经因为稳定性问题在Chrome 92版本被弃用。  -->
```

Comment:
```suggestion
**帧重启** 功能回归了! 你可以在函数中途从头开始重启当前代码行之前的代码。这个功能曾经因为稳定性问题在Chrome 92版本被弃用。
```

Reply:
Hi @xyugroup , we can remove the comment syntax on Chinese translation. This apply to the whole article.